src/components/button/utils.tsx

```diff
@@ -6,8 +6,10 @@ import { LoadingIcon } from '../loading';
 export type ButtonModes =
   | 'interactive_0'
   | 'interactive_0--bordered'
+  | 'interactive_0--bordered-light-outline'
   | 'interactive_1'
   | 'interactive_2'
+  | 'interactive_2--customized_off_palette'
```

Comment:
```suggestion
  | 'interactive_2--light-outline'
```

And the occurrences in the css file that is.